cli/src/commands/git/push.rs

```diff
@@ -393,6 +393,27 @@ pub fn cmd_git_push(
             ),
             remote = remote.as_symbol()
         );
+
+        if bookmark_updates.is_empty() {
+            let deleted_bookmarks = tx
+                .repo()
+                .view()
+                .local_remote_bookmarks(remote)
+                .filter(|(_, targets)| {
+                    !targets.local_target.is_present()
+                        && matches!(classify_bookmark_push_action(*targets),
+                        BookmarkPushAction::Update(update) if update.new_target.is_none())
+                })
+                .count();
+
+            if deleted_bookmarks > 0 {
+                writeln!(
+                    ui.hint_default(),
+                    "You can push {deleted_bookmarks} deleted bookmark{s} with `--deleted`",
+                    s = if deleted_bookmarks > 1 { "s" } else { "" }
```

Comment:
Should I add a trailing comma here?

Reply:
`rustfmt` doesn't, so on second thought, I won't; it's inconsistent with how all of the other macros are formatted throughout the codebase. (Ideally it'd be able to do it same as function calls, but I understand why it can't/doesn't.)

Reply:
Oh, I misread the comment, sorry. I meant period should be added to hint/warning messages. (Our style is a bit random about that, though.)